spec.html

```diff
@@ -7347,6 +7524,28 @@ <h1>ECMAScript Function Objects</h1>
             The <emu-xref href="#sec-source-text">source text</emu-xref> that defines the function.
           </td>
         </tr>
+        <tr>
+          <td>
+            [[Fields]]
+          </td>
+          <td>
+            List of Records as created by ClassFieldDefinitionEvaluation
+          </td>
+          <td>
+            If the function is a class with instancefield declarations, this is a list of records representing those fields and their initializers.
```

Comment:
"with instancefield declarations" reads awkwardly to me. is this a preexisting phrasing?